Libraries/acdriver/Headers/acdriver/Options.h

```diff
@@ -43,7 +44,7 @@ class Options {
 private:
     std::string _platform;
     std::string _minimumDeploymentTarget;
-    std::string _targetDevice;
+    std::set<std::string> _targetDevice;
```

Comment:
nit: I'd use `std::vector` to preserve ordering here, it's more consistent with other options structures and is useful to avoid losing information from the options parsing process. Alternatively, could use `std::unordered_set`, since I doubt this needs to be sorted.